store/datastore/utils.go

```diff
@@ -38,34 +39,33 @@ func rebind(query string) string {
 
 // helper function that converts a simple repsitory list
 // to a sql IN statment.
-func toList(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "?"
-		in[i] = repo.FullName
-	}
-	return strings.Join(qs, ","), in
-}
+func toList(listof []*model.RepoLite) ([]string, [][]interface{}) {
+	var total = len(listof)
+	var limit = 999
+	var pagesCount = int(math.Ceil(float64(total) / float64(limit)))
 
-// helper function that converts a simple repository list
-// to a sql IN statement compatible with postgres.
-func toListPosgres(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "$" + strconv.Itoa(i+1)
-		in[i] = repo.FullName
+	var qs = make([]string, pagesCount, pagesCount)
```

Comment:
shouldn't we have `[][]string` to correspond to `[][]interface{}`? `[0][]string` may have a different length than `[1][]interface{}` right? The first page, index `[0]` could have 999 items while the second page, index `[1]` could have != 999 items and therefore needs a different placeholder lenght